db/schema.rb

```diff
@@ -67,6 +74,7 @@
     t.date     "date_of_birth"
     t.string   "phone"
     t.string   "cell_phone"
+    t.string   "neighborhood"
```

Comment:
@HacksForHugs Why do users have a neighborhood string attribute? 
Remember that neighborhoods are related to users through the userNeighborhood model. 

Also, change your github so you have your name on it please lol